documentation/docs/backend/endpoints.md

```diff
@@ -60,11 +66,13 @@
 - `user_id`
 - `kilometers`
 - `date`
+- `team`
 
 #### Views
 
-- `create_mileage/` - create new mileage
-- `get_mileage/<int:user_id>` - Retrieve a list of mileage by user_id
+- `get_leaderboard/` - Returns leaderboard data
+- `get_mileage` - Retrieve a list of mileage
```

Comment:
Say "Returns" not "Retrieve" in line 74

Reply:
"Returns" not "Return", say "Returns a list of mileage data" so it matches the wording of get leaderboard